src/frontend/src/pages/ChangeRequestsPage/ChangeRequestsOverview.tsx

```diff
@@ -0,0 +1,117 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { Box, Grid, useTheme } from '@mui/material';
+import { useAllChangeRequests } from '../../hooks/change-requests.hooks';
+import LoadingIndicator from '../../components/LoadingIndicator';
+import ErrorPage from '../ErrorPage';
+import { isLeadership, isHead, ChangeRequest, Project, WorkPackage, wbsPipe } from 'shared';
+import PageBlock from '../../layouts/PageBlock';
+import { useAllProjects } from '../../hooks/projects.hooks';
+import { useAllWorkPackages } from '../../hooks/work-packages.hooks';
+import ChangeRequestDetailCard from '../../components/ChangeRequestDetailCard';
+import { useCurrentUser } from '../../hooks/users.hooks';
+
+const ChangeRequestsOverview: React.FC = () => {
+  const theme = useTheme();
+  const user = useCurrentUser();
+
+  const { isLoading, isError, error, data } = useAllChangeRequests();
+  const { data: projects, isError: projectIsError, isLoading: projectLoading, error: projectError } = useAllProjects();
+  const { data: workPackages, isError: wpIsError, isLoading: wpLoading, error: wpError } = useAllWorkPackages();
+
+  // whether to show To Review section
+  const showToReview = isHead(user.role) || isLeadership(user.role);
+
+  if (isLoading || projectLoading || wpLoading || !data || !projects || !workPackages) return <LoadingIndicator />;
+  if (isError) return <ErrorPage message={error?.message} />;
+  if (projectIsError) return <ErrorPage message={projectError?.message} />;
+  if (wpIsError) return <ErrorPage message={wpError?.message} />;
+
+  // projects whose change requests the user would have to review
+  const myProjects = projects.filter(
+    (project: Project) =>
+      (project.team ? project.team.teamId === user.teamAsLeadId : false) ||
+      (project.projectLead ? project.projectLead.userId === user.userId : false) ||
+      (project.projectManager ? project.projectManager.userId === user.userId : false)
+  );
+
+  // work packages whose change requests the user would have to review
+  const myWorkPackages = workPackages.filter(
+    (wp: WorkPackage) =>
+      (wp.projectLead ? wp.projectLead.userId === user.userId : false) ||
+      (wp.projectManager ? wp.projectManager.userId === user.userId : false)
+  );
+
+  const myWbs = myProjects
```

Comment:
As a code reader, it was hard for me to define what this variable represents. Can we give documentation on this variable? (ex. `// set of all work packages and projects to review in x.x.x string format for current user`)